package.json

```diff
@@ -34,6 +34,7 @@
     "start:storybook:test": "NODE_ENV=test start-storybook --port 57021 --quiet --ci",
     "start:storybook:dev": "NODE_ENV=development  start-storybook --port 80",
     "dev": "concurrently --raw \"yarn start:storybook:dev\" \"yarn watch\"",
+    "test:unit": "vitest",
     "test": "cypress run",
```

Comment:
I'll keep it like this for now. Since it has been OK'ed that we introduce Vitest for unit testing I would like to get it into our setup and start to write more tests. But soonish we should have configured our CI setup to make use of it also.